App/session.rb

```diff
@@ -0,0 +1,10 @@
+# class that controls the sessions of the application
+class Session
+  def initialize
+    @session_active = false
```

Comment:
Para que utilizaremos esta variable?

Reply:
#1 
```suggestion
    @session_active = false
```
@FernandoMorantes esta variable la hize por que con ella se podra usar para saber si un usuario ya inicio sesion o no. pero no se si sea necesaria